src/component/utility/toString.ts

```diff
@@ -0,0 +1,10 @@
+interface ToStringOptions {
+  trueLabel?: string;
+  falseLabel?: string;
+  negate?: boolean;
+}
+
+export function toString(flag: boolean, options: ToStringOptions = {}) {
```

Comment:
Try to find a better name because this is way too generic and too easy to confuse with the builtin JS method.